CMakeLists.txt

```diff
@@ -272,6 +272,8 @@ configure_file (
   )
 install(SCRIPT tcmu.conf_install.cmake)
 
+install(FILES libtcmu.h libtcmu_common.h tcmu-runner.h
```

Comment:
Won't the final new lib have a new header name so it does not conflict with users of the old libtcmu?

Also wether the final lib needs something like tcmu-runner.h is still up in the air.